common/src/chain/config/mod.rs

```diff
@@ -139,14 +139,14 @@ fn address_prefix(chain_type: ChainType, destination: &Destination) -> &'static
             Destination::ClassicMultisig(_) => "mmtc",
         },
         ChainType::Testnet => match destination {
-            Destination::AnyoneCanSpend => "",
+            Destination::AnyoneCanSpend => "anyonecanspend_address",
```

Comment:
Please change it for every chain type, and add a prefix signifying the net type.

Does it have to be so long, btw? Does something like `tanyonecanspend` not work?

Reply:
used the shorted form works fine as well